src/tagUtil.js

```diff
@@ -0,0 +1,45 @@
+var NAME_KEYS = [
+  'name',
+  'type',
+  'tiger:NAME'
+];
+
+/**
+ * Check if all needed tags are in place
+ *
+ * @param {object} obj
+ * @returns {boolean}
+ */
+module.exports.checkAdminTags = function checkAdminTags(obj) {
+  return obj.tags('name') &&
+    obj.tags('boundary') === 'administrative' &&
+    obj.tags('admin_level');
+};
+
+/**
+ * Concat properties to compile index id
+ *
+ * @param {object} obj
+ * @returns {string}
+ */
+module.exports.buildIndexId = function buildIndexId(obj) {
+  return obj.tags('name') + ':admin_level_' + obj.tags('admin_level');
+};
+
+/**
+ * Find name tag
+ *
+ * @param {object} obj
+ * @returns {string|undefined}
+ */
+module.exports.findNameTag = function findNameTag(obj) {
+  var name;
+  var tags = obj.tags();
+
+  NAME_KEYS.every(function (key) {
+    name = tags[key];
+    return !name; // stop if a name is found
+  });
+
+  return name;
+};
```

Comment:
Did you consider just using a straight up `for` loop here? Just my preference, but I find it a little easier to read since you don't have to deal with the indirection of being unable to `return` a value from `findNameTag()` from inside the function passed to `every()`.

```javascript
for(var ind = 0; ind < NAME_KEYS.length; ind++){
	var key = NAME_KEYS[ind];
	if(tags.hasOwnProperty(key)){
		return tags[key];
	}
}
```